.circleci/config.yml

```diff
@@ -22,18 +22,18 @@ jobs:
   build:
     executor: golang
     steps:
-      - run: go get -u github.com/prometheus/client_golang/examples/random
-      - run: ls -l /go/bin/random
+      - run: go install github.com/prometheus/client_golang/examples/random@latest
+      - run: ls -l ~/go/bin/random
       - persist_to_workspace:
-          root: /
+          root: /home/circleci
```

Comment:
https://github.com/prometheus/demo-site/blob/7db3fc6f5d71b43bf38751d32369f650f0b78a74/.circleci/config.yml#L91
https://github.com/prometheus/demo-site/blob/7db3fc6f5d71b43bf38751d32369f650f0b78a74/.circleci/config.yml#L121

Won't these have to be changed to `/home/circle/go/bin/random` then?

Reply:
No, the file is copied into the playbook, and the playbook deploys it with the correct destination.